modules/terraform/output.go

```diff
@@ -49,6 +50,135 @@ func OutputRequiredE(t testing.TestingT, options *Options, key string) (string,
 	return out, nil
 }
 
+func parseListOfMaps(l []interface{}) ([]map[string]interface{}, error) {
+	var result []map[string]interface{}
+
+	for _, v := range l {
+		m, err := parseMap(v.(map[string]interface{}))
+		if err != nil {
+			return nil, err
+		}
+		result = append(result, m)
+	}
+
+	return result, nil
+
+}
+
+// parseMap takes a map of interfaces and parses the types. Is also recursive for nested objects.
+func parseMap(m map[string]interface{}) (map[string]interface{}, error) {
+
+	var result map[string]interface{}
+	result = make(map[string]interface{})
+
+	for k, v := range m {
+		switch v.(type) {
+		case map[string]interface{}:
+			nestedMap, err := parseMap(v.(map[string]interface{}))
```

Comment:
NIT: you don't need to manually  cast `v` here. Instead, you can use a [type switch](https://tour.golang.org/methods/16) to have Go do it for you automatically (and more safely).